src/bootstrap-table.js

```diff
@@ -969,6 +974,7 @@
         if (this.options.sidePagination !== 'server') {
             var s = this.searchText && this.searchText.toLowerCase();
             var f = $.isEmptyObject(this.filterColumns) ? null : this.filterColumns;
+            var r = (typeof this.options.filterRows === "string");
```

Comment:
When we via JavaScript:
```js
bootstrapTable({
    filterRows: function () {}
});
```
Here will cause error.
BTW, please use `'` instead of `"`.